README.md

```diff
@@ -85,6 +84,10 @@ Base URL: http://stats.filspark.com/
 
   http://stats.filspark.com/deals/summary
 
+- `GET /retrieval-result-status/daily?from=2024-01-01&to=2024-01-31`
```

Comment:
I don't understand the "status" part.

In the docs (see https://www.notion.so/protocollabs/Spark-Request-Based-Non-Committee-Global-Retrieval-Success-Rate-4c5e8c47c45f467f80392d00cac2aae4?pvs=4#122837df73d4803f917bf8e8eb13f9d4), we use the term "Retrieval Result Code". 

Can we use that term for the REST API too?

```suggestion
- `GET /retrieval-result-codes/daily?from=2024-01-01&to=2024-01-31`
```

Reply:
Good idea!

Reply:
#18763e9